build/TestShared/xunit.runner.json

```diff
@@ -1,5 +1,5 @@
 {
-  "parallelizeTestCollections": true,
```

Comment:
Isn't this turning it off for all the assemblies?
Can you disable it just for NuGet.PackageManagement.Test or am I misunderstanding something.

Reply:
From the [docs](https://xunit.github.io/docs/configuring-with-json) it looks to be only for this assembly -
> Set this to true if the assembly is willing to run tests inside this assembly in parallel against each other. Tests in the same test collection will be run sequentially against each other, but tests in different test collections will be run in parallel against each other. Set this to false to disable all parallelization within this test assembly.

Reply:
This is in TestShared though...that's my question. 

Specifically why not do something like https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/NuGet.Protocol.FuncTest/xunit.runner.json